src/EditorFeatures/CSharpTest/Completion/CompletionProviders/CrefCompletionProviderTests.cs

```diff
@@ -273,7 +273,7 @@ class C { }
 /// <see cref=""List{T}.Enumerator ""/>
 class C { }
 ";
-            await VerifyProviderCommitAsync(text, "Enumerator", expected, ' ', "Enum");
+            await VerifyProviderCommitAsync(text, "Enumerator", expected, ' ');
```

Comment:
All test changes in non-LSP test files are in response to the change here: https://github.com/dotnet/roslyn/pull/45099/files#r439293576